Utility/integration-test.sh

```diff
@@ -6,6 +6,9 @@ JSON_FILES=`ls -d Examples/PDK/*.json`
 # Generate Objective-C models
 .build/debug/plank  --output_dir=Examples/Cocoa/Sources/objc/ $JSON_FILES
 
+#Generate flow types for models
+.build/debug/plank --lang flow  --output_dir=Examples/JS/flow/ $JSON_FILES
```

Comment:
We can add a `.flowconfig` to the JS repo and run `flow-bin` CLI command over the output. We need to check if `flow-bin` is available globally on the dev's machine though.